pkg/occlient/occlient_test.go

```diff
@@ -2301,6 +2358,72 @@ func TestLinkSecret(t *testing.T) {
 	}
 }
 
+func TestListSecrets(t *testing.T) {
+
+	tests := []struct {
+		name       string
+		secretList corev1.SecretList
+		output     []corev1.Secret
+		wantErr    bool
+	}{
+		{
+			name: "test case 1",
```

Comment:
Add a better name for this :) This is testing for list of secrets, making sure they are there?